hunts/src/HuntViewMain.js

```diff
@@ -2,53 +2,130 @@ import React from "react";
 import PropTypes from "prop-types";
 import { PuzzleTable } from "./puzzle-table";
 import useInterval from "@use-it/interval";
+import Badge from "react-bootstrap/Badge";
+import Button from "react-bootstrap/Button";
+import Dropdown from "react-bootstrap/Dropdown";
+import DropdownButton from "react-bootstrap/DropdownButton";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faEdit, faTrashAlt } from "@fortawesome/free-regular-svg-icons";
+import { faPlus, faTimes } from "@fortawesome/free-solid-svg-icons";
 
 const TABLE_COLUMNS = [
-  {
-    Header: "id",
-    accessor: "id",
-  },
   {
     Header: "Name",
     accessor: "name",
-    Cell: ({ row, value }) =>
-      row.canExpand ? (
-        <span
-          {...row.getToggleRowExpandedProps({
-            style: {
-              paddingLeft: `${row.depth * 2}rem`,
-            },
-          })}
-        >
-          {row.isExpanded ? "▼" : "▶"} {value}
+    Cell: ({ row, value }) => (
+      <>
+        {row.canExpand ? (
+          <span
+            {...row.getToggleRowExpandedProps({
+              style: {
+                paddingLeft: `${row.depth * 2}rem`,
+              },
+            })}
+          >
+            {row.isExpanded ? "▼" : "▶"} <b>{value}</b>
+          </span>
+        ) : (
+          <span style={{ paddingLeft: `${row.depth * 2}rem` }}>
+            <b>{value}</b>
+          </span>
+        )}{" "}
+        {row.values.is_meta ? <Badge variant="dark">META</Badge> : null}
+        <span style={{ cursor: "pointer" }}>
+          <Badge pill variant="light">
+            <FontAwesomeIcon icon={faEdit} />
+          </Badge>
+        </span>{" "}
+        <span style={{ cursor: "pointer" }}>
+          <Badge pill variant="light">
+            <FontAwesomeIcon icon={faTrashAlt} />
+          </Badge>
         </span>
-      ) : (
-        <span style={{ paddingLeft: `${row.depth * 2}rem` }}>{value}</span>
-      ),
+      </>
+    ),
   },
   {
     Header: "Answer",
     accessor: "answer",
+    Cell: ({ row, value }) =>
+      value ? (
+        <span className="text-monospace">{value}</span>
+      ) : (
+        <Button variant="outline-primary">Submit Answer</Button>
+      ),
   },
   {
     Header: "Status",
     accessor: "status",
+    Cell: ({ row, value }) =>
+      ["SOLVING", "STUCK", "EXTRACTION"].includes(value) ? (
```

Comment:
Something to consider (not sure if worthwhile or practical) -- is it possible to define these possible Status states in some common enum file and refer to the enum constants here and elsewhere instead of hardcoding strings? If we used enum constants, I wonder if we could catch typos via some static typechecking (or do we need TypeScript for that)?

Reply:
Yeah this is definitely a good idea, I'll pull some constants out to another file.

I'm supportive of migrating this to typescript but stuck with es6 initially since I haven't used typescript before, and I wanted to finish this rewrite as soon as I could since it's going to clash with other changes we want to make before mystery hunt. But I do want to make the switch maybe after mystery hunt this year.